backend/data/processors/conditions_preprocessing.py

```diff
@@ -118,16 +119,21 @@ def preprocess_conditions():
 # -------------------
 
 
-def delete_exclusions(processed):
+def delete_exclusions_and_equivalents(processed):
     """Removes exclusions from enrolment conditions"""
     # Remove exclusion string which appears before prerequisite plaintext
-    excl_string = re.search(r"(excl.*?:.*?)(pre)", processed, flags=re.IGNORECASE)
+    excl_string = re.search(r"(excl.*?:.*?)(pre|co-?req)", processed, flags=re.IGNORECASE)
+    equiv_string = re.search(r"(equivalent:.*?)(pre|co-?req)", processed, flags=re.IGNORECASE)
     if excl_string:
         processed = re.sub(excl_string.group(1), "", processed)
 
+    if equiv_string:
+        processed = re.sub(equiv_string.group(1), "", processed)
+
     # Remove exclusion string appearing after prerequisite plaintext, typically
     # at the end of the enrolment rule
-    processed = re.sub(r"excl.*", "", processed, flags=re.IGNORECASE)
+    processed = re.sub(r"((\.|,)?\s)?excl.*", "", processed, flags=re.IGNORECASE)
+    processed = re.sub(r"((\.|,)?\s)?equivalent:.*", "", processed, flags=re.IGNORECASE)
```

Comment:
hmm I have a bad feeling about this line... if the exclusion/equivalent strings appear before the prerequisite plaintext, would this remove the entire condition string? is there any edge case where equiv/excl appears before prereq for us to test? 

also, probs a good idea to use `equiv` instead of `equivalent:` since they may use `equivalence` or `equivalent courses`?

Reply:
i would be surprised if `equivalence` was used anywhere but `equivalent courses` almost certainly is used somewhere

Reply:
Yup will add some variance for that